app/models/manual.rb

```diff
@@ -0,0 +1,164 @@
+class Manual
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  attr_accessor :content_id, :base_path, :title, :summary, :body, :updated_at, :public_updated_at, :publication_state, :update_type
+
+  validates :title, presence: true
+  validates :summary, presence: true
+  validates :body, safe_html: true
+
+  def initialize(params)
+    @content_id = params.fetch(:content_id, SecureRandom.uuid)
+    @title = params.fetch(:title, nil)
+    @summary = params.fetch(:summary, nil)
+    @body = params.fetch(:body, nil)
+    @publication_state = params.fetch(:publication_state, nil)
+    @public_updated_at = params.fetch(:public_updated_at, nil)
```

Comment:
It's not turned into it by the API, but it should be sent as one. I think in this instance using the setter or the instance var will both work.